package.json

```diff
@@ -62,6 +61,7 @@
     "@typescript-eslint/eslint-plugin": "^5.0.0",
     "@typescript-eslint/parser": "^5.0.0",
     "@vercel/webpack-asset-relocator-loader": "1.7.3",
+    "appdmg": "^0.6.6",
```

Comment:
Optional dependency of:
```
└─┬ @electron-forge/maker-dmg@7.4.0
       └─┬ electron-installer-dmg@4.0.0
              └── appdmg@0.6.6 deduped
```

Since it's optional, it's not guaranteed to be included, but needed for `npm run make`. I was getting the following error without it:
```
An unhandled rejection has occurred inside Forge:
Error: Cannot find module 'appdmg'
```

Adding explicit dependency for convenience.

https://github.com/electron-userland/electron-installer-dmg/blob/ddc6db7a9a90c1cf19db1454bd41ae8a6d1fe106/package.json#L35